### PR TITLE
Add refresh control to AppNet Example

### PR DESCRIPTION
**Pull request by 3lvis**

### CODE REVIEW

File: Examples/AppNet/AppNet/ViewController.swift [hunk 1]
```diff
@@ -26,15 +26,23 @@ class ViewController: UITableViewController {
...
 
+  func setUpRefreshControl() {
+    refreshControl = UIRefreshControl()
+    refreshControl?.addTarget(self, action: "fetchNewData", forControlEvents: .ValueChanged)
+  }
+
   // MARK: Networking methods
 
   func fetchNewData() {
     networking.fetchItems { _ in
```
Comment (vadymmarkov) [inline Examples/AppNet/AppNet/ViewController.swift]: Do we need [unowned self] here?

Comment (3lvis) [inline Examples/AppNet/AppNet/ViewController.swift]: @vadymmarkov I don't know, Xcode didn't said anything, also I think you would need that if the method looked like this:

```swift
func fetchNewData() {
  self.fetchItems { _ in
    self.fetchCurrentObjects()
  }
}
```

I might be wrong.

Comment (kostiakoval) [inline Examples/AppNet/AppNet/ViewController.swift]: You don't need it. Here is why
- Cerate self ( referenceCount = 1)
- Closure capture self (+ 1 to referenceCount, = 2)
- Closure finishes executing and being deallocated, it's not saved anywhere (- 1 to referenceCount, = 1)

You need to say unowned if you don't  store the closure in the proper
https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/AutomaticReferenceCounting.html#//apple_ref/doc/uid/TP40014097-CH20-ID56

Comment (vadymmarkov) [inline Examples/AppNet/AppNet/ViewController.swift]: That's right. Thanks @kostiakoval.

**Pull request merged**